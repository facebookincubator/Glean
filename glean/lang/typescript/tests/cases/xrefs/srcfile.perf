{
  "@generated": null,
  "facts_searched": { "src.File.1": 1 },
  "full_scans": [ "src.File.1" ]
}