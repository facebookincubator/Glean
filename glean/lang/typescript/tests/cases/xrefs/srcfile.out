[
  "@generated",
  { "key": "glean/lang/typescript/tests/cases/xrefs/example.ts" },
  { "key": "global/node_modules/typescript-lsif/lib/lib.es5.d.ts" }
]